app/(tabs)/_layout.tsx

```diff
@@ -8,105 +8,102 @@ import { TabButton, TabIcon, TopTabButton, TopTabIcon } from '@/components';
 import theme from '@/constants/Colors';
 import { CreditCardIcon, FileAttachmentIcon, GroupIcon, PlusIcon } from '@/constants/Icon';
 import { GlobalContext } from '@/context/GlobalContext';
-import { GroupProvider } from '@/context/group/GroupContext';
 
 const TabLayout = () => {
   const { t } = useTranslation();
   const { colorScheme } = useColorScheme();
   const { userData } = useContext(GlobalContext);
   return (
-    <GroupProvider>
-      <Tabs
-        screenOptions={{
-          tabBarShowLabel: false,
-          tabBarStyle: {
-            borderTopWidth: 0,
-            backgroundColor: colorScheme === 'light' ? theme.sky.lightest : theme.ink.darker,
-            shadowColor: colorScheme === 'light' ? theme.ink.darkest : '',
-            shadowOffset: { width: 0, height: -2 },
-            height: 60,
+    <Tabs
+      screenOptions={{
+        tabBarShowLabel: false,
+        tabBarStyle: {
+          borderTopWidth: 0,
+          backgroundColor: colorScheme === 'light' ? theme.sky.lightest : theme.ink.darker,
+          shadowColor: colorScheme === 'light' ? theme.ink.darkest : '',
+          shadowOffset: { width: 0, height: -2 },
+          height: 60,
+        },
+      }}>
+      <Tabs.Screen
+        name="reports"
+        options={{
+          title: t('Reports'),
+          tabBarButton: (props) => {
+            return <TabButton {...props} />;
           },
-        }}>
-        <Tabs.Screen
-          name="reports"
-          options={{
-            title: t('Reports'),
-            tabBarButton: (props) => {
-              return <TabButton {...props} />;
-            },
-            headerShown: false,
-            tabBarIcon: ({ focused }) => (
-              <TabIcon name={t('Reports')} focused={focused}>
-                <FileAttachmentIcon />
-              </TabIcon>
-            ),
-          }}
-        />
-        <Tabs.Screen
-          name="groups"
-          options={{
-            title: t('Groups'),
-            tabBarButton: (props) => {
-              return <TabButton {...props} />;
-            },
-            headerShown: false,
-            tabBarIcon: ({ focused }) => (
-              <TabIcon name={t('Groups')} focused={focused}>
-                <GroupIcon />
-              </TabIcon>
-            ),
-          }}
-        />
+          headerShown: false,
+          tabBarIcon: ({ focused }) => (
+            <TabIcon name={t('Reports')} focused={focused}>
+              <FileAttachmentIcon />
+            </TabIcon>
+          ),
+        }}
+      />
+      <Tabs.Screen
+        name="groups"
+        options={{
+          title: t('Groups'),
+          tabBarButton: (props) => {
```

Comment:
could we export that function to const and pass that as function to that options ?